### PR TITLE
Add JMX Exporter Ironbank

### PR DESCRIPTION
**Pull request by Vandit1604**
### Description of the change
Adding Prometheus JMX Exporter Ironbank Image

### Applicable issues

  - fixes #

### Checklist

- [ ] Added to the `README.md` using [readme-generator](https://github.com/rapidfort/community-images/readme-generator)

### CODE REVIEW

File: community_images/jmx-exporter/ironbank/image.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: jmx-exporter-ib
+official_name: JMX Exporter IronBank
```
Comment (Rajat0002) [inline community_images/jmx-exporter/ironbank/image.yml:2]: JMX Exporter Iron Bank

Comment (Vandit1604) [reply community_images/jmx-exporter/ironbank/image.yml]: Done

File: community_images/jmx-exporter/ironbank/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+services:
+  jmx-exporter:
+    image: ${JMX_EXPORTER_IMAGE_REPOSITORY}:${JMX_EXPORTER_IMAGE_TAG}
+    ports:
+      - "12345:12345"
```
Comment (AnmolVirdi) [inline community_images/jmx-exporter/ironbank/docker-compose.yml:5]: Can we map host ports dynamically? `::12345`

**Review by AnmolVirdi — APPROVED** [review]

File: community_images/jmx-exporter/ironbank/docker-compose.yml (hunk 2 above)
Comment (Vandit1604) [reply community_images/jmx-exporter/ironbank/docker-compose.yml]: Done

Comment (Vandit1604) [reply community_images/jmx-exporter/ironbank/docker-compose.yml]: Done

**Review by AnmolVirdi — APPROVED** [review]

**Review by AnmolVirdi — APPROVED** [review]

**Pull request merged**